index.user.js

```diff
@@ -402,7 +408,8 @@ class BombAttack extends ProjectileAttack {
         if (enemy.m_bDead || EnemyWillAffectedByBoulder(enemy) || BlackholeOfEnemy(enemy) != null)
             return WORST_SCORE;
 
-        let score =  AllEnemiesHPNearPoint(enemy.m_Sprite.x, enemy.m_Sprite.y, 50);
+        let explosionWidth = GAME.m_State.m_AttackManager.m_rgExplosionFrames[0].width * 0.4; //attack.js:353 value is 204.8
```

Comment:
Nice find. Instead of `* 0.4` you could use `GAME.m_State.m_AttackManager.m_rgExplosionFrames[0].scale[1]` can you not?

Reply:
m_rgExplosionFrames[0] doesn't have the scale property:

console.log(gGame.m_State.m_AttackManager.m_rgExplosionFrames[0].scale)
undefined

Reply:
Fair enough, you would need to get the AnimatedSprite. Let's Meepen decide.
```
				var explosion = new PIXI.extras.AnimatedSprite(instance.m_rgExplosionFrames );
				explosion.animationSpeed = 0.2;
				explosion.loop = false;
				explosion.play();
				explosion.scale.set( 0.4, 0.4 );
				explosion.anchor.set( 0.5, 0.5 );
```